src/main/scala/uk/gov/hmrc/scalatestaccessibilitylinter/AccessibilityMatchers.scala

```diff
@@ -23,10 +23,14 @@ import uk.gov.hmrc.scalatestaccessibilitylinter.config.{defaultAxeLinter, defaul
 import uk.gov.hmrc.scalatestaccessibilitylinter.domain._
 
 import scala.collection.immutable.ListMap
+import scala.language.{implicitConversions, reflectiveCalls}
 
 trait AccessibilityMatchers { this: Informing =>
   protected val accessibilityLinters: Seq[AccessibilityLinter.Service] = Seq(defaultAxeLinter, defaultVnuLinter)
 
+  private type PlayLikeWrappedHtml = { def body: String }
+  implicit def playLikeHtmlToString(html: PlayLikeWrappedHtml): String = html.body
```

Comment:
I think this might have unpredictable effects on consuming service unit tests. I think the Scala community seems to be against implicit conversions like these nowadays for those reasons. Is it really that much of a deal for teams to add `.body` to the output of their Twirl templates? :-)

Reply:
It is probably a bit too generic of a conversion yes!